packages/enzyme-matchers/src/assertions/toContainMatches.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree. *
+ *
+ * @providesModule toContainMatches
+ * @flow
+ */
+
+import type { EnzymeObject, Matcher } from '../types';
+import html from '../utils/html';
+import getNodeName from '../utils/name';
+import single from '../utils/single';
+
+function toContainMatches(
+  enzymeWrapper: EnzymeObject,
+  n: Integer,
```

Comment:
is Integer a valid primitive for flow? I thought it was just `number`. Can you run `yarn flow` and see if there are errors?

Reply:
Ah, it is not. Also, String should be string.

Fixed